src/server/components/dashboard/listsItems/controllers.ts

```diff
@@ -653,72 +664,95 @@ export async function handlePublishListItem(
+  }
 }
 
 async function initialiseFormRunnerSession(
   list: List,
   listItem: BaseListItemGetObject,
   message: string,
   isUnderTest: boolean
-): Promise<string> {
-  const questions = await generateFormRunnerWebhookData(
-    list,
-    listItem,
-    isUnderTest
-  );
-  const formRunnerWebhookData = getNewSessionWebhookData(
-    list.type,
-    listItem.id,
-    questions,
-    message
-  );
-  const formRunnerNewSessionUrl = createFormRunnerReturningUserLink(list.type);
-  const token = await getInitiateFormRunnerSessionToken(
-    formRunnerNewSessionUrl,
-    formRunnerWebhookData
-  );
-  return createFormRunnerEditListItemLink(token);
+): Promise<string | undefined> {
+  try {
+    const questions = await generateFormRunnerWebhookData(
+      list,
+      listItem,
+      isUnderTest
+    );
+    const formRunnerWebhookData = getNewSessionWebhookData(
+      list.type,
+      listItem.id,
+      questions,
+      message
+    );
+    const formRunnerNewSessionUrl = createFormRunnerReturningUserLink(list.type);
+    const token = await getInitiateFormRunnerSessionToken(
+      formRunnerNewSessionUrl,
+      formRunnerWebhookData
+    );
+    return createFormRunnerEditListItemLink(token);
+  } catch (error) {
+    logger.error(`initialiseFormRunnerSession Error: ${(error as Error).message}`);
+  }
 }
 
 export async function listItemEditRequestValidation(
   req: Request,
   res: Response,
   next: NextFunction
 ): Promise<void> {
-  const { listId, listItemId } = req.params;
-  const userId = req.user?.userData?.id;
-
-  const list = await findListById(listId);
-  const listItem = await findListItemById(listItemId);
-
-  if (userId === undefined) {
-    return res.redirect(authRoutes.logout);
-  }
-
-  if (list === undefined) {
-    const err = new HttpException(404, "404", `Could not find list ${listId}`);
-    return next(err);
-
-  } else if (listItem === undefined) {
-    const err = new HttpException(404, "404", `Could not find list item ${listItemId}`);
-    return next(err);
+  try {
+    const { listId, listItemId } = req.params;
+    const userId = req.user?.userData?.id;
 
-  } else if (list?.type !== listItem?.type) {
-    const err = new HttpException(400, "400", `Trying to edit a list item which is a different service type to list ${listId}`);
-    return next(err);
+    const list = await findListById(listId);
+    const listItem = await findListItemById(listItemId);
 
-  } else if (list?.id !== listItem?.listId) {
-    const err = new HttpException(400, "400", `Trying to edit a list item which does not belong to list ${listId}`);
-    return next(err);
+    if (userId === undefined) {
+      return res.redirect(authRoutes.logout);
+    }
 
-  } else if (!userIsListPublisher(req, list)) {
-    const err = new HttpException(403, "403", "User does not have publishing rights on this list.");
-    return next(err);
+    if (list === undefined) {
+      res.status(404).send({
```

Comment:
this needs to be undone for each of these else-if blocks.  The result is that the json error message is rendered to the browser rather than the HttpException being thrown and handled by the middleware error handler.